### PR TITLE
Use iframe onload + other bugs

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- [x] Use iframe onload instead of window.onload
- [x] Fix blocks insight-chart-container analysis link title
- [x] Redirect homepape to `/wp-admin`

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: wp-content/plugins/hurumap/src/HurumapChartPreview.js [hunk 1]
```diff
@@ -26,30 +26,34 @@ function HurumapChartPreview({ chart }) {
...
-              query: geoRowCountQuery(chart.visual.table),
-              variables: {
-                geoCode: geo.geoCode,
-                geoLevel: geo.geoLevel
-              }
-            });
-            return { ...geo, totalCount: data[chart.visual.table].totalCount };
-          })
-        );
+      if (preview) {
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/HurumapChartPreview.js:32]: :100:

File: wp-content/plugins/hurumap/api.php [hunk 2]
```diff
@@ -84,6 +84,18 @@ function register_routes()
     ));
 }
 
+function get_domain()
+{
+  $url = site_url();
+  $pieces = parse_url($url);
+  $domain = isset($pieces['host']) ? $pieces['host'] : $pieces['path'];
+  if (preg_match('/(?P<domain>[a-z0-9][a-z0-9\-]{1,63}\.[a-z\.]{2,6})$/i', $domain, $regs)) {
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/api.php]: :scream: Lets just use `takwimu.africa` for now but lets have a story to add an option where someone can enter the main domain name to set to. It's hard to know what is the main/subdomain e.g. takwimu.africa, takwimu.co.tz, etc.

**Review by kilemensi — APPROVED** [review]
:rocket: @KhadijaMahanga

**Comment by KhadijaMahanga** [comment]
@kilemensi pushed analysis link title in here too

**Pull request merged**